### ISSUE TITLE
z.strictObject missing from documentation

### ISSUE THREAD

**Issue by jayfz** [colinhacks/zod#2629]
Hi,

I just went to check the docs for more information on this function and I find that is missing.

Is this intentional?

**Comment by scotttrinh** [comment]
I believe it's intentional in that the other modifiers are typically chained methods on the `ZodObject` class (`z.object({}).strict()`, `z.object({}).passthrough()`, etc). I _think_ it's just there for backwards compatibility.

**Comment by colinhacks** [comment]
Yeah it's there as a convenience but I'd prefer to point people towards `.strict()`. There will (probably) be a way to set the default `UnknownKeys` behavior in Zod 4.

**Comment by JacobWeisenburger** [comment]
Has this issue been addressed? If so, I would like to close it.
